lib/imagemounter/utils/list_developer_image.js

```diff
@@ -0,0 +1,184 @@
+import _ from 'lodash';
+import { zip, net, env, fs } from '@appium/support';
+import { join as joinPath } from 'path';
+import axios from 'axios';
+import log from '../../logger';
+const { exists, readdir, lstat, mkdir, unlink } = fs;
+/**
+ * @typedef {Object} GithubTreeObject
+ * @property {string} path
+ * @property {string} mode
+ * @property {'blob' | 'tree'} type
+ * @property {string} sha
+ * @property {number} size
+ * @property {string} url
+ */
+
+/**
+ * @typedef {Object} GithubTreeResponse
+ * @property {string} sha
+ * @property {string} url
+ * @property {GithubTreeObject[]?} tree
+ * @property {boolean?} truncated
+ * @property {string?} node_id
+ * @property {number?} size
+ * @property {string?} content
+ * @property {string?} base64
+ */
```

Comment:
regarding my treatise on `null` above, you probably mean this instead:

```suggestion
/**
 * @typedef {Object} GithubTreeResponse
 * @property {string} sha
 * @property {string} url
 * @property {GithubTreeObject[]} [tree]
 * @property {boolean} [truncated]
 * @property {string} [node_id]
 * @property {number} [size]
 * @property {string} [content]
 * @property {string} [base64]
 */
```